dimensiondata/src/test/java/org/jclouds/dimensiondata/cloudcontrol/features/NetworkApiLiveTest.java

```diff
@@ -104,14 +98,6 @@ public void testDeploySameNetworkDomain() {
             NetworkApiLiveTest.class.getSimpleName(), "ESSENTIALS");
    }
 
-   @Test(dependsOnMethods = "testDeployVlan")
-   public void testAddPublicIpBlock() {
```

Comment:
I had to remove this live test as this function cannot be tested until we have a server deployed. We have some validation that prevents addition of public ip blocks until there is at least one server.